docs/components/stepper/index.html

```diff
@@ -67,7 +65,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Issue Details</span>
               <span class="beta-hxStepValue"></span>
             </header>
-            <div class="hxBox hxMd">
+            <div class="hxBox hxMd beta-hxContainerStyle">
```

Comment:
We shouldn't need the extra class.  Please remove.

Reply:
Changes have been done, please review.